wdfn-server/waterdata/tests/test_views.py

```diff
@@ -36,6 +36,16 @@ def test_version(self):
         self.assertEqual(response.status_code, 200)
 
 
+class TestIvDataAvailabilityStatementView(TestCase):
+    def setUp(self):
+        self.app_client = app.test_client()
+
+    def test_version(self):
```

Comment:
Probably should call this test something else (and the one for the Provisional Data statement view). Maybe test_successful_view ... we aren't testing the version.

Reply:
Very true. Changed.